### PR TITLE
Add OpenTelemetry Example

### PR DESCRIPTION
**Pull request by NikitaSkrynnik**
## Issue link

https://github.com/networkservicemesh/sdk/issues/584

## How Has This Been Tested?

- [ ] Added unit testing to cover
- [ ] Tested manually
- [ ] Tested by integration testing
- [ ] Have not tested

## Types of changes

- [ ] Bug fix
- [ ] New functionallity
- [ ] Documentation
- [ ] Refactoring
- [ ] CI

### CODE REVIEW

File: apps/jaeger/jaeger.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
```
Comment (denis-tingaikin) [inline apps/jaeger/jaeger.yaml:1]: We store only applications that are based on NSM SDK in the folder apps/. Please move this near the example.

File: apps/jaeger/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
```
Comment (denis-tingaikin) [inline apps/jaeger/kustomization.yaml:1]: We store only applications that are based on NSM SDK in the folder apps/. Please move this near the example.

File: apps/opentelemetry/kustomization.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
```
Comment (denis-tingaikin) [inline apps/opentelemetry/kustomization.yaml:1]: We store only applications that are based on NSM SDK in the folder apps/. Please move this near the example.

File: apps/opentelemetry/otel-collector.yaml [hunk 4]
```diff
@@ -0,0 +1,108 @@
+---
```
Comment (denis-tingaikin) [inline apps/opentelemetry/otel-collector.yaml:1]: We store only applications that are based on NSM SDK in the folder apps/. Please move this near the example.

File: apps/prometheus/kustomization.yaml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+---
```
Comment (denis-tingaikin) [inline apps/prometheus/kustomization.yaml:1]: We store only applications that are based on NSM SDK in the folder apps/. Please move this near the example.

File: apps/prometheus/prometheus.yaml [hunk 6]
```diff
@@ -0,0 +1,67 @@
+---
```
Comment (denis-tingaikin) [inline apps/prometheus/prometheus.yaml:1]: We store only applications that are based on NSM SDK in the folder apps/. Please move this near the example.

**Review by denis-tingaikin — COMMENTED** [review]
Other things look valid. Please move deployments for observability near observability examples

**Comment by denis-tingaikin** [comment]
Also why is this still draft?

File: apps/jaeger/jaeger.yaml (hunk 1 above)
Comment (glazychev-art) [inline apps/jaeger/jaeger.yaml:1]: Please remove duplicate files

**Comment by denis-tingaikin** [comment]
@NikitaSkrynnik Please apply review comments.

**Comment by NikitaSkrynnik** [comment]
@denis-tingaikin @glazychev-art Fixed

**Review by glazychev-art — APPROVED** [review]

**Review by denis-tingaikin — APPROVED** [review]

**Pull request merged**